### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by YHerm**
removed static in intake and funnel and inserted to simulate and neo package
removed final from sysid

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 1]
```diff
@@ -30,10 +30,14 @@ private OI() {
...
-    public static OI getInstance() {
-        if (instance == null) {
+    public static void init(){
+        if (instance == null){
             instance = new OI();
         }
+    }
+
+    public static OI getInstance() {
+        init();
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/OI.java:40]: this is reserved for when clean-master is merged, return it to what it was

File: src/main/java/edu/greenblitz/robotName/Robot.java [hunk 2]
```diff
@@ -44,13 +46,18 @@ public void robotInit() {
         initializeSubsystems();
         SwerveChassis.getInstance().resetAllEncoders();
         initializeAutonomousBuilder();
-        OI.getInstance();
+        OI.init();
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/Robot.java:49]: same here

File: src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeFactory.java [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
-			case REPLAY:
-				return new ReplayIntake();
-			case SIMULATION:
-			default:
-				return new SimulationIntake();
-		}
+        return switch (RobotConstants.ROBOT_TYPE) {
+            case ROBOT_NAME -> new NeoIntake();
+            case REPLAY -> new ReplayIntake();
+			case SIMULATION -> new SimulationIntake();
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeFactory.java:12]: why is it so far

**Review by YoniKiriaty — CHANGES_REQUESTED** [review]
almost

**Review by YoniKiriaty — APPROVED** [review]

**Review by aaa-png — APPROVED** [review]
UNICODE: 4f000

File: src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeFactory.java (hunk 3 above)
Comment (tnrz135) [reply src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeFactory.java:12]: Why was it resolved without being fixed

**Review by tnrz135 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

**Review by tnrz135 — APPROVED** [review]

**Review by itamaroryan — APPROVED** [review]
Good stuff

**Comment by bossyshrimp68** [comment]
המחליק הוא המצחיק

**Pull request merged**